lobster/tools/core/html_report.py

```diff
@@ -461,6 +461,9 @@ def main():
                     default="report.lobster")
     ap.add_argument("--out",
                     default="lobster_report.html")
+    ap.add_argument("--dot",
+                    help="path to dot utility, by default expected in PATH",
```

Comment:
just a minor comment: not everybody knows what "dot" is, and these users might be confused what this help message means. A google search for "dot" does not immediately point to `https://graphviz.org/`. How about the following message:
`help="path to dot utility (https://graphviz.org/), by default expected in PATH",`

Reply:
done